packages/web-components/src/components/ic-select/ic-select.tsx

```diff
@@ -134,7 +134,7 @@ export class Select {
   @Prop() validationText?: string = "";
 
   /**
-   * The value of the currently selected option.
+   * The value of the select, reflected by the value of the currently selected option.
```

Comment:
I'm not sure if this covers all bases because a searchable value may not come a selected option.